src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -21,11 +21,21 @@ const clone = rfdc();
 const DEFAULT_DATE_FORMAT = 'yyyy-MM-dd';
 const MAX_ERRORS_COUNT = 100;
 
+export const _getColumnWithoutDepth = (columns) => {
+  return columns
+    .flatMap((columnOrColumnGroup) => {
+      if (columnOrColumnGroup == null)
+        console.warn('Null or undefined values found in columns list, please check the solution configuration');
+      return columnOrColumnGroup?.children ? _getColumnWithoutDepth(columnOrColumnGroup.children) : columnOrColumnGroup;
+    })
+    .filter((columns) => columns);
```

Comment:
why do we need this filter function that filters nothing?

Reply:
He do not filter nothing. It's a lambda function consist of exclude null or undefined value and then, fix unstable columns definition or crash in the webapp